kong/plugins/datadog/schema.lua

```diff
@@ -95,7 +96,17 @@ return {
                     if_match = { one_of = { "counter", "gauge" }, },
                     then_field = "sample_rate",
                     then_match = { required = true },
-                  }, },
-  }, }, }, }, }, }, }, },
+          }, }, }, }, }, },
+        },
+        entity_checks = {
+          { conditional = {
+            if_field = "use_env", if_match = {eq = false},
```

Comment:
@javierguerragiraldez I don't like the name `use_env` but also don't know of a better alternative, thoughts?

Reply:
The use case here is defining the host and port when DataDog is run as a `daemonset` under Kubernetes.  In theory, this configuration could be applied more broadly though